src/main/resources/assets/computercraft/lua/rom/programs/edit.lua

```diff
@@ -59,6 +59,7 @@ end
 if peripheral.find( "printer" ) then
     table.insert( tMenuItems, "Print" )
 end
+table.insert( tMenuItems, "Jump" )
```

Comment:
Not a fan of adding another item to the menu. It complicates the UI of one of the earliest things new CC users see. The beginner flow is very important